src/Artsy/Router/Boot.tsx

```diff
@@ -57,32 +57,32 @@ export class Boot extends React.Component<BootProps> {
     }
 
     return (
-      <ErrorBoundary>
+      <Theme>
```

Comment:
Was it necessary to move the existing error boundary in order to get this to work? Are there any tradeoffs in having it as the outermost wrapper vs having it inside the system context provider?

Reply:
This was just so that I could use Theme file inside the error boundary, which shows the nav. 

> Are there any tradeoffs in having it as the outermost wrapper vs having it inside the system context provider

The only trade off i can see is that an error thrown from within the render function of the context provider proper -- not its children -- wouldn't get caught by it, but there isn't too much logic there. We might want to think about it in the future if we decide to expand functionality at that level.